Lesson 2/app/src/main/res/layout/activity_fragment.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I guess this file is for fragment's sample. So the name of the file could be better.